pycroft/lib/user.py

```diff
@@ -433,6 +438,24 @@ def suspend(user, reason, processor, during=None):
 
 
 @with_transaction
+def unblock(user, processor):
+    """Unblocks a user.
+
+    This removes his membership of the ``config.violation`` group.
+
+    :param User user: The user to be unblocked.
+    :param User processor: The admin who unblocked the user.
+
+    :return: The unblocked user.
+    """
+    remove_member_of(user=user, group=config.violation_group,
```

Comment:
This is a bug, without giving `during=closedopen(when, None)` this removes every membership in the group retroactively instead of terminating.